### PR TITLE
G33 minor change

### PR DESCRIPTION
**Pull request by LVD-AC**
-further named remaining tests
-A and O parameters obsolete
-quick homing with home_delta()
-preventing premature end

### CODE REVIEW

File: Marlin/Marlin_main.cpp [hunk 1]
```diff
@@ -5082,7 +5078,7 @@ void home_all_axes() { gcode_G28(); }
 
       const int8_t verbose_level = code_seen('V') ? code_value_byte() : 1;
       if (!WITHIN(verbose_level, 0, 2)) {
-        SERIAL_PROTOCOLLNPGM("?(V)erbose level is implausible (0-2).");
+        SERIAL_PROTOCOLLNPGM("?(V)erbose Level is implausible (0-2).");
```
Comment (bgort) [inline Marlin/Marlin_main.cpp]: This was changed in a previous merge, and it isn't important really, but it suggests a rebase may be needed?

Comment (LVD-AC) [reply Marlin/Marlin_main.cpp]: I copied the whole G33 routine from a non-Git local copy on my drive I use to develop and beta test. Hence not 100% up to date with the latest merges....

Comment (bgort) [reply Marlin/Marlin_main.cpp]: Yeah, no worries.  Just better to not undo recent changes, if possible.

Comment (LVD-AC) [reply Marlin/Marlin_main.cpp]: done

**Comment by bgort** [comment]
FWIW, it looks good to me.. but I don't have a delta so can't test.

**Comment by LVD-AC** [comment]
tested and all is well

**Comment by LVD-AC** [comment]
Squashed into #6700

**Pull request closed without merging**